token/client/src/token.rs

```diff
@@ -259,85 +287,116 @@ where
         ))))
     }
 
-    pub async fn process_ixs<S2: Signers>(
+    pub async fn construct_tx<S2: Signers>(
```

Comment:
nit: for this and `process_ixs`, since we've removed the `S` type on the struct, we can rename `S2` to `S`